pipeline_dash/viz/dash/components/graph_col.py

```diff
@@ -60,4 +71,27 @@ def generate(app: dash.Dash, graph: networkx.DiGraph) -> Tuple[dbc.Col, go.Figur
         lg=7,
         xs=12,
     )
+
+    @app.callback(
+        OperatorOutput(Ids.graph, "figure"),
+        Input(Ids.stores.show_annotations, "data"),
+        State(viz_dash.Ids.stores.session_id, "data"),
+        State(viz_dash.Ids.stores.figure_root, "data"),
+        prevent_initial_call=False,
+    )
+    @logged_callback
+    # @pcprofile
```

Comment:
dead code, maybe intentional? There's several of these, but I'll leave just the one comment

Reply:
these are semi-intentional to make it easy to turn profiling back on for some critical callbacks. Leaving this for now